src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -45,6 +45,40 @@ public async Task SendAsync_UserAgent_CorrectlyWritten()
             });
         }
 
+        [Fact]
+        public async Task SendAsync_LargeHeaders_CorrectlyWritten()
+        {
+            // Intentionally larger than 16K in total because that's the limit that will trigger a CONTINUATION frame in HTTP2.
+            string largeHeaderValue = new string('a', 1024);
+            int count = 20;
+
+            await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+            {
+                using HttpClient client = CreateHttpClient();
+
+                var message = new HttpRequestMessage(HttpMethod.Get, uri) { Version = UseVersion };
+                for (int i = 0; i < count; i++)
+                {
+                    message.Headers.TryAddWithoutValidation("large-header" + i, largeHeaderValue);
+                }
+                var response = await client.SendAsync(TestAsync, message).ConfigureAwait(false);
+            },
+            async server =>
+            {
+                HttpRequestData requestData = await server.HandleRequestAsync(HttpStatusCode.OK);
+
+                if (requestData.Path == "/setup")
```

Comment:
What's this test for?

Reply:
Nothing, forgotten copypasta, good catch though :+1: